applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -174,7 +176,12 @@ def Initialize(self):
     def InitializeSolutionStep(self):
         if self._TimeBufferIsInitialized():
             (self.bdf_process).Execute()                    # Recompute the BDF2 coefficients
-            (self.level_set_convection_process).Execute()   # Convect the level-set according to the previous step velocity
+
+            if not self._bfecc_convection:
```

Comment:
Why not the other way around?  "if self._bfecc_convection:" ...  It looks a little unnatural to me.

Reply:
Ok, I agree.